client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -147,27 +143,15 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 		};
 
 		submitForm = () => {
-			const {
-				fields,
-				jetpackFieldsToUpdate,
-				jetpackSiteSettingsAPIVersion,
-				settingsFields,
-				siteId,
-				siteIsJetpack,
-				jetpackSettingsUISupported,
-			} = this.props;
+			const { fields, jetpackFieldsToUpdate, settingsFields, siteId, siteIsJetpack } = this.props;
 			this.props.removeNotice( 'site-settings-save' );
 			debug( 'submitForm', { fields, settingsFields } );
 
-			// Support site settings for older Jetpacks as needed
-			const siteFields = pick( fields, settingsFields.site );
-			const apiVersion = siteIsJetpack ? jetpackSiteSettingsAPIVersion : '1.4';
-
-			if ( jetpackSettingsUISupported ) {
+			if ( siteIsJetpack ) {
 				this.props.saveJetpackSettings( siteId, jetpackFieldsToUpdate );
 			}
 
-			this.props.saveSiteSettings( siteId, { ...siteFields, apiVersion } );
+			this.props.saveSiteSettings( siteId, pick( fields, settingsFields.site ) );
```

Comment:
What happened to the `apiVersion` field? Until now, it defaulted to 1.4, but if not specified, it will default to 1.1: https://github.com/Automattic/wp-calypso/blob/master/client/lib/wpcom-undocumented/lib/undocumented.js#L353

Reply:
Oops, that's a great catch!

Restored in bbb288ed8fabdc9e5c2bff7f886e60c8ae56e9e3